modules/rubiconBidAdapter.js

```diff
@@ -150,14 +139,14 @@ export const spec = {
       let slotData = {
         site_id: params.siteId,
         zone_id: params.zoneId,
-        position: parsePosition(params.position),
+        position: params.position === 'atf' || params.position === 'btf' ? params.position : 'unknown',
```

Comment:
why undo this?  I think this logic was purposely moved into `parsePosition` since its shared with the `p_pos` logic below.

Reply:
I'm trying to follow openRTB standard and remove parsePosition for video: after the video endpoint migrated to PBS Java openRTB, the new endpoint runs auction with [OpenRTB 2.5](https://www.iab.com/wp-content/uploads/2016/03/OpenRTB-API-Specification-Version-2-5-FINAL.pdf) bid request, which uses integer values for ads position.